src/styles/components/_content-type.scss

```diff
@@ -31,3 +31,7 @@
 .c-content-type--video .o-svg-icon {
 	color: $color-yellow;
 }
+
+.c-content-type--assignment .o-svg-icon {
+	color: $color-ocean-green;
```

Comment:
The icon will have to be “French rose” #F33F67
so it matches with task: https://meemoo.atlassian.net/browse/AVO-2637